modin/core/io/io.py

```diff
@@ -174,42 +174,9 @@ def read_json(
         summary="Load data from Google BigQuery into query compiler",
         returns=_doc_returns_qc,
     )
-    def read_gbq(
-        cls,
-        query: str,
-        project_id=None,
-        index_col=None,
-        col_order=None,
-        reauth=False,
-        auth_local_webserver=False,
-        dialect=None,
-        location=None,
-        configuration=None,
-        credentials=None,
-        use_bqstorage_api=None,
-        private_key=None,
-        verbose=None,
-        progress_bar_type=None,
-        max_results=None,
-    ):  # noqa: PR01
+    def read_gbq(cls, query: str, **kwargs):  # noqa: PR01
```

Comment:
I don't like that you're changing API in a PR marked as `TEST-`.

Plus this change hurts IDE hinting and interactive help...

Reply:
reverted